### ISSUE TITLE
[Issue] Lots of chat spam from CUSTOM_TRENCH effect

### ISSUE THREAD

**Issue by johnyuki** [GC-spigot/AdvancedEnchantments#1779]
**Describe the bug**
Sometimes when players are using the Detonate enchantment that we have modified to use CUSTOM_TRENCH, it will spam that chat with errors for seemingly no reason. The enchantment still works as shown in the video below, and no one has ever complained about the enchantment not working. This is what we get in the console when it happens:
```
[17:24:41 WARN]: java.lang.NullPointerException
[17:24:41 INFO]: [AdvancedEnchantments ERROR] Could not pass NullPointerException: 
[17:24:41 INFO]: [AdvancedEnchantments]    Class: 
[17:24:41 INFO]: [AdvancedEnchantments]    Extra info: effect[CUSTOM_TRENCH:1, 3, 1]; enchant [detonate]; mc[116]; ae[8.0.0-Pre20.9]
[17:24:41 INFO]: [AdvancedEnchantments] If you cannot indentify cause of this, contact developer providing this report. 
```
Here is our Detonate config:
```
detonate:
  display: '%group-color%Detonate'
  description: "&fChance to break in 1x3x1 area.\n&nMax level: 9"
  applies-to: 'Pickaxes'
  type: 'MINING'
  group: 'HIGH'
  applies:
    - ALL_PICKAXE
    - ALL_SPADE
  levels:
    1:
      chance: 13
      effects:
        - 'CUSTOM_TRENCH:1:3:1'
    2:
      chance: 26
      effects:
        - 'CUSTOM_TRENCH:1:3:1'
    3:
      chance: 36
      effects:
        - 'CUSTOM_TRENCH:1:3:1'
    4:
      chance: 49
      effects:
        - 'CUSTOM_TRENCH:1:3:1'
    5:
      chance: 59
      effects:
        - 'CUSTOM_TRENCH:1:3:1'
    6:
      chance: 72
      effects:
        - 'CUSTOM_TRENCH:1:3:1'
    7:
      chance: 85
      effects:
        - 'CUSTOM_TRENCH:1:3:1'
    8:
      chance: 90
      effects:
        - 'CUSTOM_TRENCH:1:3:1'
    9:
      chance: 100
      effects:
        - 'CUSTOM_TRENCH:1:3:1'
```

**To Reproduce**
Unknown. I have literally no idea why this happens. 

**Screenshots / Video**
https://streamable.com/s8rkwr

**Server Information (please complete the following information):**
- Server Version: Purplane version git-Purplane-"e4752a5"
- Plugin Version: 8.0.0-pre-20.9 (we have since updated and it still happens)

**Comment by Insprill** [comment]
Please update to 21.1 and see if it still happens

**Comment by Insprill** [comment]
Fixed in the next update

**Comment by Insprill** [comment]
Will be fixed in 21.4, still present in 21.3.
